metadata/metadata.go

```diff
@@ -81,12 +81,12 @@ func Pairs(kv ...string) MD {
 	return md
 }
 
-// Len returns the number of items in md.
+// Len returns the number of items in MD.
```

Comment:
The usage is inconsistent. In line 130 and several other places, MD is used. I'll change all of them to md then.